x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -128,21 +131,12 @@ export const tasks: TelemetryTask[] = [
             index: indicesByProcessorEvent[processorEvent],
             body: {
               size: 0,
+              timeout,
```

Comment:
I can't comment directly on this piece since you didn't change it but can you rewrite it slightly?
<img src="https://user-images.githubusercontent.com/209966/87006722-7391b800-c1c1-11ea-9bbe-f06fa9280811.png" width="500"/>


To something like this perhaps:


```ts
interface Job {
  processorEvent: ProcessorEvent;
  timeRange: TimeRange;
}

const events = Object.keys(indicesByProcessorEvent) as ProcessorEvent[];
const jobs: Jobs[] = events.flatMap((processorEvent) =>
  TIME_RANGES.map((timeRange) => ({ processorEvent, timeRange }))
);
```

Would make it easier to digest